Dockerfile.dev

```diff
@@ -9,5 +9,9 @@ COPY . /opt/saltcorn
 RUN lerna bootstrap
 
 ENV PATH "$PATH:/opt/saltcorn/packages/saltcorn-cli/bin/saltcorn"
+ENV SALTCORN_MULTI_TENANT "true"
```

Comment:
We shouldn't set configuration environment variables here, otherwise they will apply to everyone using this docker image - this image is used for anyone who wants to deploy the up-to-date bleeding edge Saltcorn between releases.